### ISSUE TITLE
Why not add a password save function

### ISSUE THREAD

**Issue by x0penghui** [connectbot/connectbot#668]
# Why not add a password save function
# I think a good software is because it is human enough, the ability to save passwords is very convenient, no need to be safe, I just want to connect some servers simply and quickly. #
# I hope that you can add this feature for simplicity. #

**Comment by alescdb** [comment]
An easy way is to genererate a key without password (eg. with `ssh-keygen`).
This will solve your problem.

**Comment by andy5995** [comment]
@x0penghui What I did was

1. Using connectbot generate a public key
2. Copied public key to clipboard
3. ssh into host
4. open ~/.ssh/authorized_keys
5. paste the public key into the text file and save

Then you can connect to the host without needing a password at all, and the pass will never be transmitted over the network.

**Comment by alexisdal** [comment]
yes it's a pitty to feel kicked out of good open source software. 

@paolo-ienne thanks for the tip on juiceSSH that does implement the feature.

**Comment by KaKi87** [comment]
Duplicate of #355

**Comment by TarcisioMenezes** [comment]
+1 for this
Agree that a key can be used instead, but an option to save the password is also useful.
The ones that don't need this feature will have the option to not use it. I'm pretty sure this would be useful for a lot of people.

**Comment by AdamSchuttovits** [comment]
Same for me, came from JuiceSSH, and unfortunately going back after 15 minutes because of this.

**Comment by wedok** [comment]
Hmm, 7 years later ... ... ...
I also tried "username:password@ip:port" but still asking for password.
Is this so unbelievable hard to implement?
:)

**Comment by CHN-MuXin** [comment]
> Hmm, 7 years later ... ... ... I also tried "username:password@ip:port" but still asking for password. Is this so unbelievable hard to implement? :)

This issue is not a matter of difficulty; rather, it lies in the author's philosophy.

**Comment by kruton** [comment]
In the latest beta.

**Comment by paolo-ienne** [comment]
Thanks!

**Comment by batata004** [comment]
@kruton  i followd the link of the merge that was proposed to fix this password save issue, but i cant find the beta release apk so i can download. could you please tell me where is it located?
